poo/atv_campus_ufc/CampusLista.py

```
@@ -0,0 +1,129 @@
+            return
+        if not nome_curso in self.campus[nome_campus]:
+            print("esse curso não foi cadastrado nesse campus.")
+            return
+        
+        self.campus[nome_campus].remove(nome_curso)
+        print(f"o curso {nome_curso} foi removido do campus {nome_campus}.")
+        
+        
+    def atualizar_campus(self, campus: str, novo_nome_campus: str):
+        campus = campus.lower()
+        if not campus in self.campus:
```

Ruff (0.14.4)

Test for membership should be `not in`

Convert to `not in`

(E713)